src/activity_item_handling.cpp

```diff
@@ -361,13 +380,14 @@ static void move_items( const tripoint &source_arg, const tripoint &destination_
 
             // Drop it first since we're going to delete the original.
             dropped_items.push_back( *temp_item );
-            g->drop( dropped_items, dropped_worn, 0, destination.x, destination.y );
+            // no 3d function yet
```

Comment:
Why a re you calling it with a tripoint as argument when it's not a 3d function?